css/responsive/mobile.styl

```diff
@@ -0,0 +1,59 @@
+.site-title
+  text-align: center
+  &.color-white
```

Comment:
i think better for these kind of active states on a overlay or whatever to add a body class with js and style from there. more readable and can be used if needed later down the line